optuna/storages/base.py

```diff
@@ -201,16 +201,6 @@ def get_trial_system_attrs(self, trial_id):
 
     # Methods for PercentilePruner and MedianPruner
 
-    def get_best_intermediate_result_over_steps(self, trial_id):
```

Comment:
It might be better to make this method deprecated because of this is public.

Reply:
> I guess there is no user who calls this method.

I agree with you. It seems okay to simply remove this method.